src/styles/blocks/card.scss

```
@@ -0,0 +1,89 @@
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  background: #fff;
+  color: $firstTextColor;
+  &__img {
```

Add the blank line between the selectors
```suggestion
  color: $firstTextColor;
  
  &__img {
```

---

Not fixed